make_queue/templates/make_queue/quota/quota_user.html

```diff
@@ -23,4 +23,11 @@
             {% endif %}
         </div>
     </div>
-{% endfor %}
\ No newline at end of file
+{% endfor %}
+
+{% if user_quotas|length == 0 %}
```

Comment:
Use djangos [for-empty](https://docs.djangoproject.com/en/2.1/ref/templates/builtins/#for-empty) instead.